### PR TITLE
Features

### PR DESCRIPTION
**Pull request by sahilda**

### CODE REVIEW

File: TaskHero/TaskDetailViewController.swift [hunk 1]
```diff
@@ -117,12 +122,46 @@ extension TaskDetailViewController: UITableViewDelegate, UITableViewDataSource {
 extension TaskDetailViewController: TaskInstanceUpdateDelegate {
     
     func taskInstanceUpdated() {
+        if taskInstance?.getPercentComplete() == 1.0 {
```
Comment (jcomo) [inline TaskHero/TaskDetailViewController.swift]: Might want to do >= 1.0 here because of floating point math

**Comment by jcomo** [comment]
Minor comment. Looks good otherwise

**Comment by akishinagawa** [comment]
Looks good to me too.

**Comment by sahilda** [comment]
Thanks for the review earlier - made a few changes.
* fixed the percentage @jcomo.
* added a common file that we can use setting colors: AppColors.swift. The idea being to use these colors throughout (based on @akishinagawa's selection) so that if we need to, we'll be able to change it easily.
* updated Navigation controller colors throughout
* Added the ability to save / load images to specific steps.

features

**Comment by akishinagawa** [comment]
Looks good and works good.

**Pull request merged**